src/components/Auth/loginComponent.js

```diff
@@ -46,7 +44,7 @@ const IndexPage = ({ classes }) => {
       <>
         { (isSignedIn) ? (
           <>
-            <AfterSignInComponent userName={userName} signoutLink={signOut} />
+            <AfterSignInComponent userName={firstName || email} signoutLink={signOut} />
```

Comment:
@ogwuazors This was added in separate PR. It's overwritten section.  It bringing up old changes.